test/e2e-tests.sh

```diff
@@ -78,6 +78,9 @@ if (( HTTPS )); then
   add_trap "turn_off_auto_tls" SIGKILL SIGTERM SIGQUIT
 fi
 
+# Enable allow-zero-initial-scale before running e2e tests (for test/e2e/initial_scale_test.go)
+kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-autoscaler --type=merge --patch='{"data":{"allow-zero-initial-scale":"true"}}'
```

Comment:
Do we need to register traps as well like the autotls tests?

Reply:
Even broader question: Should we just allow this for tests anyway? It shouldn't harm all the other tests :man_shrugging:

Reply:
Right it shouldn't harm the other tests, so we could just leave this on and not reset it.